### PR TITLE
Set autoDark as default until user changes the theme

### LINKED ISSUE #14: If userChoice and autoDark are activated, the system color is no longer automatically used.
If the functions userChoice and autoDark are activated you can toggle the design only with the button. The system setting is no longer used as the default.

Can the autoDark setting be the default until Toggle design is changed by a user?

### PR DESCRIPTION
**Pull request by Gamemap**
Please check if this is correct, because this is my first pull request and I don't have good knowledge about javascript and HTML.

fixes #14

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -95,16 +95,33 @@ jQuery(function() {
     if (jQuery('#configUserChoice').attr('content') == '0'){
         localStorage.setItem('configUserChoice', '0');
     }
+	
+	if (jQuery('#configAutoDark').attr('content') == '1'){
+        localStorage.setItem('configAutoDark', '1');
+		localStorage.setItem('changedAutoDark', '0');
```
Comment (MrReSc) [inline script.js:101]: Unfortunately, it does not work for me. changedAutoDark is set back to 0 every time a new page is loaded.

**Review by MrReSc — COMMENTED** [review]
Can you test and revise the whole thing again?
Thanks for your collaboration on the theme.

File: script.js (hunk 1 above)
Comment (Gamemap) [reply script.js:102]: I think it is now resolved.

**Comment by Gamemap** [comment]
Please wait to merge, because I want to clean up the code a bit.

**Comment by MrReSc** [comment]
Yes, there's no hurry. I also want to fix the flicker problem #8.

**Comment by Gamemap** [comment]
> Yes, there's no hurry. I also want to fix the flicker problem #8.

I am working on this too (almost finished).

**Comment by MrReSc** [comment]
Oh, that's good to hear. Have you understood the problem, how the flickering comes about?

**Comment by Gamemap** [comment]
The flickering is caused by the theme used in the admin settings before it loads the JS. So I set the default theme to `autoadmin` (a value that is not used), test for it in the JS and change the theme.

I think it is not the nicest way to do this, but it is almost working.

**Comment by MrReSc** [comment]
Can you tell me what you did with your change? The flickering is still the same for me. 
That the auto mode is kept until the user has made a change, does not work for me either.
Maybe we should handle the flickering and the new function in two separate PR.

**Comment by Gamemap** [comment]
> Maybe we should handle the flickering and the new function in two separate PR.

I agree with you on this.


> The flickering is still the same for me.

The problem was that my computer loaded the pages from the cache, so it stopped flickering and I thought it was fixed. Maybe we should delete that and start over.

Unfortunately, I don't have much time to work on it until Jannuar.

**Comment by MrReSc** [comment]
I am keeping Issue #8 and #14 open. You can work on it at any time. I am grateful for any help.

**Comment by Gamemap** [comment]
@MrReSc Could you try if this is working?
https://github.com/Gamemap/MindTheDark/tree/Gamemap-Set_autoDark_as_default_until_user_changes_the_theme

**Comment by MrReSc** [comment]
Can you create a new PR? Then I can see your changes more easily.

**Pull request closed without merging**